src/main/scala/rocket/ICache.scala

```diff
@@ -448,13 +448,16 @@ class ICacheModule(outer: ICache) extends LazyModuleImp(outer)
     val user_bit_cacheable = edge_out.manager.supportsAcquireTFast(refill_paddr, lgCacheBlockBytes.U)
 
     x.privileged  := io.privileged || user_bit_cacheable    // privileged if machine mode or memory port
-    x.cacheable   := user_bit_cacheable
+
+    // enable outer caches for cacheable address PMAs
+    x.bufferable  := user_bit_cacheable
+    x.modifiable  := user_bit_cacheable
+    x.readalloc   := user_bit_cacheable
+    x.writealloc  := user_bit_cacheable
```

Comment:
I did NOT tie these all to true. I think that is a debate for another PR.

Reply:
Since travis failed anyway, I made this change, since it sounds like we now agree this is the right way to go.